compiler/one-cmds/one-build

```diff
@@ -154,25 +161,26 @@ def main():
     config = _parse_cfg(args)
 
     # verify configuration file
-    drivers = [
-        'one-import-tf', 'one-import-tflite', 'one-import-bcq', 'one-import-onnx',
-        'one-optimize', 'one-quantize', 'one-pack', 'one-codegen'
-    ]
+    driver_types = [_utils.DriverType.IMPORTER, _utils.DriverType.MISC]
+    drivers = _collect_available_drivers(driver_types)
     _verify_cfg(drivers, config)
 
     # verify optimization option file
     _verify_opt(args)
 
-    # get sections to run
-    section_to_run = []
-    for d in drivers:
-        if _is_available_driver(config, d):
-            section_to_run.append(d)
+    # get drivers to run
+    driver_to_run = []
+    for driver_path in drivers:
+        driver_spec = drivers[driver_path]
+        driver_name = driver_spec[0]
+        if _is_available_driver(config, driver_name):
+            driver_to_run.append(driver_path)
 
```

Comment:
@binarman As @mhs4670go noticed in https://github.com/Samsung/ONE/issues/7962#issuecomment-965983975, drivers can be launched in wrong order. In case of *one-build_001.test* `one-optimize` is first element in `driver_to_run` and `one-import-tf` is second.

I think sorting of `driver_to_run` in right order could fix this PR.

Reply:
Now I noticed this thing too, now I understand why we needed explicit driver list =)